Composer/packages/lib/indexers/src/dialogIndexer.ts

```diff
@@ -187,6 +188,7 @@ function parse(id: string, content: any) {
     lgTemplates: extractLgTemplates(id, content),
     referredLuIntents: extractLuIntents(content, id),
     luFile: getBaseName(luFile, '.lu'),
+    qnaFile: getBaseName(qnaFile, '.lu.qna') || getBaseName(qnaFile, '.lu'),
```

Comment:
I think if a dialog use qna, it would be: 
1. `"recognizer": "qnasample-0.lu.qna",`
2. `"recognizer": "qnasample-0.qna",`

should not be
3. `"recognizer": "qnasample-0.lu",`

So here maybe change to below ?
```
qnaFile: getBaseName(qnaFile, '.lu.qna') || getBaseName(qnaFile, '.qna'),
```

Reply:
@alanlong9278

Reply:
the correct format is `${dialogName}.lu.qna`, but the old bot maybe `${dialogName}.lu`, so I add ` qnaFile: getBaseName(qnaFile, '.lu.qna') || getBaseName(qnaFile, '.lu'),` for migrating with old bot.

Reply:
This format like `${dialogName}.qna` is not going to  happen.